index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+
+'use strict';
+
+const fs = require('fs');
+
+const filePaths = [
+  __dirname + '/data/first.txt',
+  __dirname + '/data/second.txt',
+  __dirname + '/data/third.txt'
+];
+
+var result = [];
+var count = 0;
+
+filePaths.forEach(function(path, index){
+  fs.readFile(path, function(err, data){
```

Comment:
Well done, I like that you've made this more modular by incorporating it into a forEach loop.